test.py

```diff
@@ -15,17 +15,21 @@ def func_exists(mod_path, func_name):
     try:
         with open(mod_path, "r") as file:
             module_content = file.read()
-    except FileNotFoundError:
+    except FileNotFoundError as e:
+        warnings.warn(f"SKIPPED: {mod_path},{func_name}: \n{e}")
         return False  # Module file not found
 
     try:
         tree = ast.parse(module_content)
-    except SyntaxError:
+    except SyntaxError as e:
+        warnings.warn(f"SKIPPED: {mod_path},{func_name}: \n{e}")
```

Comment:
According to the comment in the next line of code, maybe:

msg = f"SKIPPED: Syntax error in module. \n{e}"
warnings.warn(msg)